lib/twemoji.rb

```diff
@@ -172,10 +203,11 @@ def self.filter_emojis(content)
     # @return [String] Emoji image tag generated by name and options.
     # @private
     def self.img_tag(name)
+      unicode = render_unicode(name)
       if options[:img_attr]
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{name}' #{options[:img_attr]} src='#{emoji_url(name)}'>"
+        "<img class='#{options[:class_name]}' draggable='false' alt='#{unicode}' #{options[:img_attr]} src='#{emoji_url(name)}'>"
```

Comment:
You know I added title back, but to be honest I feel like it should be something one adds via `img_attrs` since it is different than the default upstream in the js lib.